File: MORM.Java/src/MORM/Java/Models/Venda/Pessoa.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "PESSOA")
public class Pessoa extends CollectionItem
{
    @CampoAttribute(Campo = "ID_PESSOA", Tipo = CampoTipo.Key)
    public String Id_Pessoa;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_PESSOA", Tipo = CampoTipo.Req)
    public Integer Cd_Pessoa;
    @CampoAttribute(Campo = "NM_PESSOA", Tipo = CampoTipo.Req)
    public String Nm_Pessoa;
    @CampoAttribute(Campo = "NR_CPFCNPJ", Tipo = CampoTipo.Req)
    public String Nr_Cpfcnpj;
    @CampoAttribute(Campo = "NR_RGIE", Tipo = CampoTipo.Req)
    public String Nr_Rgie;
    @CampoAttribute(Campo = "NM_FANTASIA", Tipo = CampoTipo.Nul)
    public String Nm_Fantasia;
    @CampoAttribute(Campo = "CD_CEP", Tipo = CampoTipo.Req)
    public Integer Cd_Cep;
    @CampoAttribute(Campo = "NM_LOGRADOURO", Tipo = CampoTipo.Req)
    public String Nm_Logradouro;
    @CampoAttribute(Campo = "NR_LOGRADOURO", Tipo = CampoTipo.Req)
    public String Nr_Logradouro;
    @CampoAttribute(Campo = "DS_BAIRRO", Tipo = CampoTipo.Req)
    public String Ds_Bairro;
    @CampoAttribute(Campo = "DS_COMPLEMENTO", Tipo = CampoTipo.Nul)
    public String Ds_Complemento;
    @CampoAttribute(Campo = "CD_MUNICIPIO", Tipo = CampoTipo.Req)
    public Integer Cd_Municipio;
    @CampoAttribute(Campo = "DS_MUNICIPIO", Tipo = CampoTipo.Req)
    public String Ds_Municipio;
    @CampoAttribute(Campo = "CD_ESTADO", Tipo = CampoTipo.Req)
    public Integer Cd_Estado;
    @CampoAttribute(Campo = "DS_ESTADO", Tipo = CampoTipo.Req)
    public String Ds_Estado;
    @CampoAttribute(Campo = "DS_SIGLAESTADO", Tipo = CampoTipo.Req)
    public String Ds_Siglaestado;
    @CampoAttribute(Campo = "CD_PAIS", Tipo = CampoTipo.Req)
    public Integer Cd_Pais;
    @CampoAttribute(Campo = "DS_PAIS", Tipo = CampoTipo.Req)
    public String Ds_Pais;
    @CampoAttribute(Campo = "DS_FONE", Tipo = CampoTipo.Nul)
    public String Ds_Fone;
    @CampoAttribute(Campo = "DS_CELULAR", Tipo = CampoTipo.Nul)
    public String Ds_Celular;
    @CampoAttribute(Campo = "DS_EMAIL", Tipo = CampoTipo.Nul)
    public String Ds_Email;
    @CampoAttribute(Campo = "IN_CONSUMIDORFINAL", Tipo = CampoTipo.Nul)
    public String In_Consumidorfinal;
}

File: MORM.CSharp/MORM.CSharp/Models/Venda/Pessoa.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("PESSOA")]
    public class Pessoa : CollectionItem
    {
        [Campo("ID_PESSOA", CampoTipo.Key)]
        public string Id_Pessoa { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_PESSOA", CampoTipo.Req)]
        public int Cd_Pessoa { get; set; }
        [Campo("NM_PESSOA", CampoTipo.Req)]
        public string Nm_Pessoa { get; set; }
        [Campo("NR_CPFCNPJ", CampoTipo.Req)]
        public string Nr_Cpfcnpj { get; set; }
        [Campo("NR_RGIE", CampoTipo.Req)]
        public string Nr_Rgie { get; set; }
        [Campo("NM_FANTASIA", CampoTipo.Nul)]
        public string Nm_Fantasia { get; set; }
        [Campo("CD_CEP", CampoTipo.Req)]
        public int Cd_Cep { get; set; }
        [Campo("NM_LOGRADOURO", CampoTipo.Req)]
        public string Nm_Logradouro { get; set; }
        [Campo("NR_LOGRADOURO", CampoTipo.Req)]
        public string Nr_Logradouro { get; set; }
        [Campo("DS_BAIRRO", CampoTipo.Req)]
        public string Ds_Bairro { get; set; }
        [Campo("DS_COMPLEMENTO", CampoTipo.Nul)]
        public string Ds_Complemento { get; set; }
        [Campo("CD_MUNICIPIO", CampoTipo.Req)]
        public int Cd_Municipio { get; set; }
        [Campo("DS_MUNICIPIO", CampoTipo.Req)]
        public string Ds_Municipio { get; set; }
        [Campo("CD_ESTADO", CampoTipo.Req)]
        public int Cd_Estado { get; set; }
        [Campo("DS_ESTADO", CampoTipo.Req)]
        public string Ds_Estado { get; set; }
        [Campo("DS_SIGLAESTADO", CampoTipo.Req)]
        public string Ds_Siglaestado { get; set; }
        [Campo("CD_PAIS", CampoTipo.Req)]
        public int Cd_Pais { get; set; }
        [Campo("DS_PAIS", CampoTipo.Req)]
        public string Ds_Pais { get; set; }
        [Campo("DS_FONE", CampoTipo.Nul)]
        public string Ds_Fone { get; set; }
        [Campo("DS_CELULAR", CampoTipo.Nul)]
        public string Ds_Celular { get; set; }
        [Campo("DS_EMAIL", CampoTipo.Nul)]
        public string Ds_Email { get; set; }
        [Campo("IN_CONSUMIDORFINAL", CampoTipo.Nul)]
        public string In_Consumidorfinal { get; set; }
    }
}